### PR TITLE
Replace cross-reference in .NET pages

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Changed xref to this page on RS256: https://auth0.com/blog/rs256-vs-hs256-whats-the-difference/
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* n/a

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/code/dotnet/jwt-validation/index.md](https://github.com/okta/okta-developer-docs/blob/49374d497c9c2bfe199bb3b400bae8adbf2f2614/packages%2F%40okta%2Fvuepress-site%2Fcode%2Fdotnet%2Fjwt-validation%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/40ba5085-0bc4-43ca-8d34-029ddcd82b6d) | 100  | 100  | 56  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/dotnet/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/49374d497c9c2bfe199bb3b400bae8adbf2f2614/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fdotnet%2Faccesstoken.md) | [74](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1e20e548-b30c-47ab-bd03-6f6329eeda9c) | 87  | 69  | 34  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.aa4f363e-7a6c-4076-a09c-d13015d5497e?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/code/dotnet/jwt-validation/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ This guide uses the official Microsoft OpenID Connect and JWT libraries, but you
 
 ### Get the signing keys
 
-Okta signs JWTs using [asymmetric encryption (RS256)](https://stackoverflow.com/a/39239395/3191599), and publishes the public signing keys in a JWKS (JSON Web Key Set) as part of the OAuth 2.0 and OpenID Connect discovery documents. The signing keys are rotated on a regular basis. The first step to verify a signed JWT is to retrieve the current signing keys.
+Okta signs JWTs using [asymmetric encryption (RS256)](https://auth0.com/blog/rs256-vs-hs256-whats-the-difference/), and publishes the public signing keys in a JWKS (JSON Web Key Set) as part of the OAuth 2.0 and OpenID Connect discovery documents. The signing keys are rotated on a regular basis. The first step to verify a signed JWT is to retrieve the current signing keys.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/code/dotnet/jwt-validation/index.md:33]:
```suggestion
Okta signs JWTs using [asymmetric encryption (RS256)](https://auth0.com/blog/rs256-vs-hs256-whats-the-difference/), and publishes the public signing keys in a JSON Web Key Set (JWKS) as part of the OAuth 2.0 and OpenID Connect discovery documents. The signing keys are rotated on a regular basis. The first step to verify a signed JWT is to retrieve the current signing keys.
```

File: packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/dotnet/accesstoken.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ For additional details about the code described here, see [.NET JWT Validation G
 
 ### Get the signing keys
 
-Okta signs JWTs using [asymmetric encryption (RS256)](https://stackoverflow.com/a/39239395/3191599), and publishes the public signing keys in a JWKS (JSON Web Key Set) as part of the OAuth 2.0 and OpenID Connect discovery documents. The signing keys are rotated on a regular basis. The first step to verify a signed JWT is to retrieve the current signing keys.
+Okta signs JWTs using [asymmetric encryption (RS256)](https://auth0.com/blog/rs256-vs-hs256-whats-the-difference/), and publishes the public signing keys in a JWKS (JSON Web Key Set) as part of the OAuth 2.0 and OpenID Connect discovery documents. The signing keys are rotated on a regular basis. The first step to verify a signed JWT is to retrieve the current signing keys.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/dotnet/accesstoken.md:7]:
```suggestion
Okta signs JWTs using [asymmetric encryption (RS256)](https://auth0.com/blog/rs256-vs-hs256-whats-the-difference/), and publishes the public signing keys in a JSON Web Key Set (JWKS) as part of the OAuth 2.0 and OpenID Connect discovery documents. The signing keys are rotated on a regular basis. The first step to verify a signed JWT is to retrieve the current signing keys.
```

File: packages/@okta/vuepress-site/code/dotnet/jwt-validation/index.md
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/code/dotnet/jwt-validation/index.md:1]: Is JWT defined somewhere before this paragraph?

**Review by barbaravo-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/code/dotnet/jwt-validation/index.md](https://github.com/okta/okta-developer-docs/blob/d0712b319c2d400e34946f47d09feae8322351e3/packages%2F%40okta%2Fvuepress-site%2Fcode%2Fdotnet%2Fjwt-validation%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c5cab9d5-6901-412c-b961-0d5bdafbeab0) | 100  | 100  | 56  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/dotnet/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/d0712b319c2d400e34946f47d09feae8322351e3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fdotnet%2Faccesstoken.md) | [74](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e92608e9-fef8-4a9a-80a9-e4983aee6436) | 87  | 69  | 34  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.89d73716-1fd3-4015-bf9c-f10aa40dec75?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/code/dotnet/jwt-validation/index.md](https://github.com/okta/okta-developer-docs/blob/0c4a27d35b6b29e0e6441465ea9fa7dc9b44752c/packages%2F%40okta%2Fvuepress-site%2Fcode%2Fdotnet%2Fjwt-validation%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4b61e01d-1eed-4e42-b3cd-3669f7ebf20d) | 100  | 100  | 56  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/dotnet/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/0c4a27d35b6b29e0e6441465ea9fa7dc9b44752c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fdotnet%2Faccesstoken.md) | [74](https://okta.acrolinx.cloud/api/v1/checking/scorecards/979af921-b700-4bea-996f-7ca1098747fb) | 87  | 69  | 34  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.7420577a-9030-4ccd-a866-263545349e56?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/code/dotnet/jwt-validation/index.md
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/code/dotnet/jwt-validation/index.md:1]: Kinda, will require an update. One second.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/code/dotnet/jwt-validation/index.md](https://github.com/okta/okta-developer-docs/blob/ed59cc8ff121bad6dc63ec88c2525de51c596f87/packages%2F%40okta%2Fvuepress-site%2Fcode%2Fdotnet%2Fjwt-validation%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/652d4bac-123e-496c-a1c9-85e67b8c73b7) | 100  | 100  | 56  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/dotnet/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/ed59cc8ff121bad6dc63ec88c2525de51c596f87/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fdotnet%2Faccesstoken.md) | [74](https://okta.acrolinx.cloud/api/v1/checking/scorecards/dcd626d9-8e40-4ce9-819c-3c0aefdf9590) | 87  | 69  | 34  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/index.md](https://github.com/okta/okta-developer-docs/blob/ed59cc8ff121bad6dc63ec88c2525de51c596f87/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Findex.md) | [68](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e1bdd5cc-9e29-40b9-b74d-13688d8ef6f6) | 81  | 89  | 55  | 79  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.b0dbc019-45cc-4489-bb94-a3aa24aaaa72?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Pull request merged**